src/main/webapp/mapsScript.js

```diff
@@ -25,10 +25,10 @@ var directionsRenderer;
 
 // Attach callback function to the `window` object
 window.initMap = function() {
-  // Manhattan coords
-  const coords = {lat: 40.771, lng: -73.974};
+  // initialize coords
+  const coords = {lat: 0, lng: 0};
```

Comment:
The built in DirectionsRenderer re-centers/zooms the map so that the entire route is shown, so I just initialize the map to (0, 0) and it adjusts when I call `directionsRenderer.setDirections(response)`